packages/manager/modules/pci/src/projects/project/storages/databases/database/general-information/general-information.html

```diff
@@ -0,0 +1,298 @@
+                    >
+                        <oui-tile-description>
+                            <span
+                                data-translate="pci_databases_general_information_maintenance_window"
+                                data-translate-values="$ctrl.database.maintenanceWindow"
+                            ></span>
+                        </oui-tile-description>
+                    </oui-tile-definition>
+                    <oui-tile-definition
+                        data-term="{{:: 'pci_databases_general_information_private_network_label' | translate }}"
+                    >
+                        <oui-tile-description>
+                            <span
+                                data-ng-if="!$ctrl.privateNetwork"
+                                data-translate="pci_database_common_none"
+                            ></span>
+                            <div data-ng-if="$ctrl.privateNetwork">
+                                <p
+                                    class="mb-0"
+                                    data-ng-bind=":: $ctrl.privateNetwork.id"
+                                ></p>
+                                <p
+                                    class="mb-0"
+                                    data-ng-bind=":: $ctrl.subnet.name"
+                                ></p>
+                            </div>
+                        </oui-tile-description>
+                    </oui-tile-definition>
+                    <oui-tile-definition
+                        data-term="{{:: 'pci_databases_general_information_allowed_ips_label' | translate }}"
+                    >
+                        <oui-tile-description>
+                            <span
+                                data-ng-if="$ctrl.allowedIps.length === 0"
+                                data-translate="pci_database_common_none"
+                            ></span>
+                            <p
+                                class="mb-0"
+                                data-ng-if="$ctrl.allowedIps.length !== 0"
+                                data-ng-repeat="allowedIp in $ctrl.allowedIps track by allowedIp.ip"
+                                data-ng-bind="allowedIp.ip"
+                            ></p>
+                            <button
+                                class="oui-tile__button oui-button oui-button_ghost oui-button_block"
+                                data-ng-click="$ctrl.goToAllowedIPs()"
+                            >
+                                <span
+                                    data-translate="pci_databases_general_information_manage_ips"
+                                ></span>
+                                <i
+                                    class="oui-icon oui-icon-arrow-right ml-1"
+                                    aria-hidden="true"
+                                ></i>
+                            </button>
```

Comment:
why not use oui-button?

Reply:
I could not get the required look and feel using `oui-button`. For example, the padding is not required here. But we will not be able to remove the padding...